Fix FTBFS on sparc, should be sent upstream (on my todo list)

--- qemu-0.14.1+dfsg.orig/sparc.ld
+++ qemu-0.14.1+dfsg/sparc.ld
@@ -37,8 +37,20 @@ SECTIONS
   .rela.fini     : { *(.rela.fini)      }
   .rel.bss       : { *(.rel.bss)                }
   .rela.bss      : { *(.rela.bss)               }
-  .rel.plt       : { *(.rel.plt)                }
-  .rela.plt      : { *(.rela.plt)               }
+  .rel.plt      :
+  {
+    *(.rel.plt)
+    PROVIDE_HIDDEN (__rel_iplt_start = .);
+    *(.rel.iplt)
+    PROVIDE_HIDDEN (__rel_iplt_end = .);
+  }
+  .rela.plt       :
+  {
+    *(.rela.plt)
+    PROVIDE_HIDDEN (__rela_iplt_start = .);
+    *(.rela.iplt)
+    PROVIDE_HIDDEN (__rela_iplt_end = .);
+  }
   .init          : { *(.init)   } =0x47ff041f
   .text      :
   {
